aspnet/toc.yml

```diff
@@ -1,5 +1,5 @@
 - name: ASP.NET documentation
-  href: /aspnet/#pivot=aspnet
+  href: https://docs.microsoft.com/en-us/aspnet/core/
```

Comment:
```suggestion
  href: /aspnet/core/
```

Reply:
@scottaddie, I thought we already determined that we had to use complete URL's for across repos, so I intentionally used them here.  I'll accept the change anyway and we can test again.  We had tested that on the hub page early on you might remember and could not get it to work.